### PR TITLE
Log in which subfolder a git command get's executed

### PR DESCRIPTION
**Pull request by baurmatt**

### CODE REVIEW

File: bin/git-bulk [hunk 1]
```diff
@@ -70,7 +70,7 @@ function guardedExecution () {
 
 # atomic git command execution with log
 function atomicExecution () {
-  echo 1>&2 "${bldred}->${reset} executing ${inverse}git $gitcommand${reset}" && git "$@"
+  echo 1>&2 "${bldred}->${reset} executing in ${gitrepodir} ${inverse}git $gitcommand${reset}" && git "$@"
```
Comment (spacewander) [inline bin/git-bulk:73]: It always shows `-> executing in . git summary` no matter what the git repo is.

Comment (baurmatt) [reply bin/git-bulk:73]: That's strange 🤔 It works works for me:

```bash
$ mkdir foo && cd foo
$ git clone https://github.com/tj/git-extras
$ git clone https://github.com/git/git
$ git clone https://github.com/hashicorp/terraform
$ git bulk --addworkspace test $PWD
$ git bulk status
Core command "status" accepted.
-> executing in ./terraform git status
On branch main
Your branch is up to date with 'origin/main'.

nothing to commit, working tree clean
-> executing in ./git git status
On branch master
Your branch is up to date with 'origin/master'.

nothing to commit, working tree clean
-> executing in ./git-extras git status
On branch master
Your branch is up to date with 'origin/master'.

nothing to commit, working tree clean
```

Comment (spacewander) [reply bin/git-bulk:73]: But it doesn't work for me:

```
git bulk -a log --oneline -n 1
Core command "log" accepted.
-> executing in . git log --oneline -n 1
7653b48 (HEAD -> pr/1011) Log in which subfolder a git command get's executed
-> executing in . git log --oneline -n 1
6da667d (HEAD -> master, origin/master, origin/HEAD) ww
```

Comment (baurmatt) [reply bin/git-bulk:73]: I will take a look at it in the new year.

**Comment by guenhter** [comment]
I guess this one will be covered by https://github.com/tj/git-extras/pulls

**Comment by baurmatt** [comment]
@guenhter Which one? ;)

**Comment by guenhter** [comment]
Soooryyy. copy paste error: https://github.com/tj/git-extras/pull/1020

**Comment by baurmatt** [comment]
Ah nice! :) Closing this one than :)

**Pull request closed without merging**